.gitignore

```diff
@@ -1,3 +1,5 @@
 .DS_Store
 vendor
 composer.lock
+.idea
```

Comment:
Hello, can you remove this change from `.gitignore` file?
It's really specific to you environment and you can put it to your global `.gitignore` file.

Thanks!